packages/tracing/src/span.ts

```diff
@@ -169,6 +169,18 @@ export class Span implements SpanInterface {
 
     childSpan.transaction = this.transaction;
 
+    if (__DEBUG_BUILD__ && childSpan.transaction) {
+      const opStr = (spanContext && spanContext.op) || '< unknown op >';
+      const nameStr = childSpan.transaction.name || '< unknown name >';
+      const idStr = childSpan.transaction.spanId;
+
+      // TODO: It would be safer to shove this in metadata (would need to move `metadata` from the `Transaction` class
+      // to the `Span` class)
```

Comment:
Right, can we do this easily? Though I wonder if this is a good idea because it opens up the possibility to add stuff in `span.metadata` and we'd have to make sure that this is removed in all spans before the transaction is sent to Sentry.

Reply:
Actually... Now that I think about it, I can use the transaction metadata for this purpose. Stand by...

Reply:
Update: Done.